src/helpers/results/send.auth.js

```diff
@@ -0,0 +1,10 @@
+const sendResult = (res, status, message, user, token) => res.status(status).json({
+  message,
+  user: {
+    email: user.email,
+    token,
+    username: user.name
```

Comment:
@alainburindi do we want to send this as 'username'? I'm thinking the name should reflect what the field represents. Except if we've made a design decision to refer to the user's name as username all through the app.

Reply:
I've made the changes to reflect the reality